warehouse/query-core/src/test/java/datawave/query/testframework/QueryJexl.java

```diff
@@ -0,0 +1,454 @@
+                    Assert.assertNotNull(entry);
+                    if (entry instanceof ASTStringLiteral || entry instanceof ASTNumberLiteral) {
+                        // exp is "value OP field"
+                        // remove op node
+                        SimpleNode opNode = nodes.removeFirst();
+                        normalizeField(id.image, (JexlNode) entry, opNode);
+                    } else {
+                        // push entry back on stack and add identifier
+                        nodes.addFirst(entry);
+                        nodes.addFirst(child);
+                    }
+                } else if (child instanceof ASTStringLiteral || child instanceof ASTNumberLiteral) {
+                    // check for identifier on node stack
+                    SimpleNode entry = nodes.removeFirst();
+                    Assert.assertNotNull(entry);
+                    if (entry instanceof ASTIdentifier) {
+                        // exp is "field OP value"
+                        SimpleNode opNode = nodes.removeFirst();
+                        normalizeField(((JexlNode) entry).image, (JexlNode) child, opNode);
+                    } else {
+                        // push entry back on stack and add literal
+                        nodes.addFirst(entry);
+                        nodes.addFirst(child);
+                    }
+                }
+            }
+        }
+    }
+    
+    private void normalizeField(final String field, final JexlNode value, final SimpleNode opNode) {
+        Assert.assertNotNull(opNode);
+        Normalizer<?> norm = this.manager.getNormalizer(field);
+        if (null != norm) {
+            if (norm instanceof NumberNormalizer) {
+                try {
+                    Integer.parseInt(value.image);
+                } catch (NumberFormatException nfe) {
+                    // don't normalize
+                    return;
+                }
+            }
+            // check for regex nodes
+            if (opNode instanceof ASTERNode || opNode instanceof ASTNRNode) {
+                if (!(norm instanceof NumberNormalizer)) {
+                    value.image = norm.normalizeRegex(value.image);
+                }
+            } else {
+                value.image = norm.normalize(value.image);
+            }
+        }
+    }
+    
+    /**
+     * Test class used to create an expression from the normalized JEXL script. <br>
+     * NOTE: The query string wil not be changed but the script will represent the normalized expression.
+     */
+    private static class NormalizedExpression extends ExpressionImpl {
+        NormalizedExpression(JexlEngine engine, String query, ASTJexlScript script) {
+            super(engine, query, script);
+        }
+        
+    }
+    
+    // =============================
+    // classes for testing purposes
+    public static class QueryJexlTest {
+        
+        private static final int TestEntries = 4;
```

Comment:
standard java naming convention would make this TEST_ENTRIES instead of TestEntries (which looks like a class name)

Reply:
renames to TEST_ENTRIES